flexget/tests/api_tests/test_pending_list_api.py

```diff
@@ -167,6 +167,82 @@ def test_pending_list_entries(self, api_client, schema_match):
         errors = schema_match(OC.pending_list_entry_base_object, data)
         assert not errors
 
+    def test_pending_list_entries_batch(self, api_client, schema_match):
```

Comment:
i think it'd be better to split this into several tests

Reply:
done